frontend/src/components/pages/CampOverview/EditCamperModal.tsx

```diff
@@ -28,7 +29,7 @@ const EditCamperModal = ({
       <ModalContent>
         <ModalHeader>Modal Title</ModalHeader>
         <ModalCloseButton />
-        <ModalBody>{camperId}</ModalBody>
+        <ModalBody>{camper.firstName}</ModalBody>
```

Comment:
Since camper can be undefined, would we need `camper?.firstName`? I know this is an incomplete modal but just wanted your thoughts since I'm doing the same thing in my modal.

Reply:
I don't think the camper can be undefined. The props only allow for Camper type. I check the state of `selectedCamper` before passing it in as a prop